system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -117,4 +117,11 @@ system_metrics_collector::ProcCpuData LinuxCpuMeasurementNode::makeSingleMeasure
   }
 }
 
+void LinuxCpuMeasurementNode::publishStatisticMessage()
+{
+  auto msg = generateStatisticMessage(get_name(), "cpu_usage", window_start_,
```

Comment:
cpu_usage should most likely be a static const and should be system_cpu_usage.

Reply:
Done.